.projenrc.js

```diff
@@ -5,7 +5,12 @@ const project = new GemeenteNijmegenCdkApp({
   devDeps: ['@gemeentenijmegen/projen-project-type'],
   name: 'haal-centraal-brp',
 
-  deps: ['@gemeentenijmegen/aws-constructs'], /* Runtime dependencies of this module. */
+  deps: ['@gemeentenijmegen/aws-constructs',
+    '@gemeentenijmegen/utils',
```

Comment:
Ben er wat later bij, maar zijn deze packages echt nodig?
Is https niet node native en is de keuze voor het niet gebruiken van native fetch bewust?
Als wrap ansi 6 nodig is, handig om erbij te vermelden waarom, dan kunnen we proberen die te verwijderen bij nieuwe versies van de package die dit nodig zou hebben

Reply:
HTTPS is niet genoeg om de certificaten af te handelen. Maar hier gaat sowieso nog wat in wijzigen. Zie commentaar van Joost.

Betreft ansi hier moet ik nog eens naar kijken, dit ging een keer fout bij een nieuwe versie waardoor ik geforceerd terug moest naar v6